src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java

```diff
@@ -49,13 +52,39 @@ public void remindPasswordFromLoginDropdown() {
     dropdown.verifyUserLoggedIn(userName);
   }
 
+  @Test()
+  @Execute(onWikia = "agas")
```

Comment:
this test is not dependent on wiki data - it tests core functionality that always should work the same way regardless of wiki content.